api-report/tree2.api.md

```diff
@@ -1917,6 +1920,16 @@ type TreeSchemaSpecification = [
 FlattenKeys<(StructSchemaSpecification | MapSchemaSpecification | LeafSchemaSpecification) & Partial<StructSchemaSpecification & MapSchemaSpecification & LeafSchemaSpecification>>
 ][_InlineTrick];
 
+// @alpha
+export enum TreeStatus {
+    // (undocumented)
+    Deleted = 2,
+    // (undocumented)
+    InDocument = 0,
+    // (undocumented)
+    Removed = 1
+}
```

Comment:
put the doc comment for each status on the enum member itself. This will improve intelisense fo them, and will make them not show up as undocumented here.

Reply:
Updated doc comment for each status on enum